plasmapy/__init__.py

```diff
@@ -38,7 +38,19 @@
 # Packages may add whatever they like to this file, but
 # should keep this content at the top.
 # ----------------------------------------------------------------------------
-from .version import __version__
+try:
+    # ./version.py is generated by setuptools_scm during install
+    from plasmapy.version import __version__
+except ModuleNotFoundError:
+    # package is not installed
+    from warnings import warn
+
+    warn('plasmapy.__version__ not generated, PlasmaPy is not an installed package.',
```

Comment:
"... or `setuptools_scm` is not installed."

Reply:
Honestly, I think we should require `setuptools_scm` or have a fallback version so whenever `plasmapy` is installed it has a version tag.  How to do this is another question. I'll have to think about it.

Reply:
Since we've defined `setup_requires=setuptools_scm`, `setuptools_scm` will be downloaded for the install and stored in the `./egg` as per `setuptools` documentation.

> setup_requires
>
>    A string or list of strings specifying what other distributions need to be present in order for the setup script to run. setuptools will attempt to obtain these (using pip if available) before processing the rest of the setup script or commands. This argument is needed if you are using distutils extensions as part of your build process; for example, extensions that process setup() arguments and turn them into EGG-INFO metadata files.
>
>    (Note: projects listed in setup_requires will NOT be automatically installed on the system where the setup script is being run. They are simply downloaded to the ./.eggs directory if they’re not locally available already. If you want them to be installed, as well as being available when the setup script is run, you should add them to install_requires and setup_requires.)